### PR TITLE
jenkins: List OS version in AMI test job

### PR DESCRIPTION
**Pull request by cgwalters**
Otherwise one has to back up to the cloud job to extract it when
browsing, or drop to the AWS API to list the tags.

### CODE REVIEW

**Comment by cgwalters** [comment]
(untested)

File: Jenkinsfile.cloud [hunk 1]
```diff
@@ -253,6 +253,7 @@ node(NODE) {
     def ami_intermediate = utils.sh_capture("jq -r .HVM ${dirpath}/aws-${AWS_REGION}-smoketested.json")
     build job: 'coreos-rhcos-aws-test', wait: false, parameters: [
         string(name: 'ami_intermediate', value: "${ami_intermediate}"),
+        string(name: 'version', value: "${version}"),
```
Comment (dm0-) [inline Jenkinsfile.cloud]: It looks like these parameters aren't defined in the AWS job's properties.  Is the server logging security warnings about them?

Comment (cgwalters) [reply Jenkinsfile.cloud]: Offhand I don't see any, though I don't know the specific place to look for that.

Comment (dm0-) [reply Jenkinsfile.cloud]: On the Container Linux Jenkins masters, they'd be in the `jenkins.service` journal output.

Comment (cgwalters) [reply Jenkinsfile.cloud]: I don't see any warnings of that form.

Comment (jlebon) [reply Jenkinsfile.cloud]: It looks like we're running with `-Dhudson.model.ParametersAction.keepUndefinedParameters=true`, which is... not good. We'd have to ask the other tenants of this Jenkins why that is; I distinctly remember having to deal with this in the past. Though I'd agree it'd be better to explicitly add them as parameters.

Comment (jlebon) [reply Jenkinsfile.cloud]: So, I can turn this on in the next master restart. It looks like this is the only job that would need fixing, so we can adapt it first and then turn it on.

**Pull request merged**